rilldata/rill
# Cloud UI: better Share Organization CTA

## Checklist

## Summary
#### Issue addressed: 
Third step in this [PRD](https://www.notion.so/rilldata/Make-dashboard-links-more-prominent-in-Cloud-UI-b6869e020a8d45528bfbda44aa2b6e56)

#### Details:
This PR follows #2892 and #2929. This PR:
- Creates a "Share" call-to-action specifically for Organization Admins. Organization Viewers will not see this CTA.
- Removes the Share CTA on the homepage, which was ambiguous because users could have >1 organization on the homepage.

## Steps to Verify
1. Look at the home page and see that there's no more Share CTA
2. Look at an org page and see the new Share CTA for Admins and the removal of the CTA for Viewers
